input/examples/content/assets/images/assetId/assetSource/GET/Asset.xml.example

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Asset media-type="application/vnd.ez.api.Asset+xml" href="/api/ezp/v2/content/assets/images/UtrE5DcgEyg/unsplash">
+    <uri>https://images.unsplash.com/photo-1544568100-847a948585b9?ixlib=rb-1.2.1&amp;ixid=eyJhcHBfaWQiOjE1MDk5MH0</uri>
+    <assetId>UtrE5DcgEyg</assetId>
+    <assetSource>unsplash</assetSource>
+    <assetMetadata>
```

Comment:
I think we should either mention or omit metadata as it comes from unsplash integration at it will vary with different services.